install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -4268,3 +4268,46 @@ authentication:
 enableInternalTrafficPolicy: true
 # -- Enable LoadBalancer IP Address Management
 enableLBIPAM: true
+# -- Standalone DNS Proxy Configuration
+standaloneDnsProxy:
+  # -- Enable standalone DNS proxy(alpha feature)
+  enabled: false
+  # -- Roll out Standalone Dns proxy automatically when configmap is updated.
+  rollOutPods: false
+  # -- L7 proxy mode enables proxy for DNS traffic 
+  l7Proxy: true
```

Comment:
What's the purpose of this value? Why not use the Cilium one, if the two values have to match?